docs/components/component/model1.md

```diff
@@ -75,8 +75,8 @@ Edit and fill in the attributes as applicable.
 
 The following attributes are available for `model1` arms:
 
```

Comment:
Do you also want a `<!-- prettier-ignore -->` here too?

Reply:
```suggestion

<!-- prettier-ignore -->
```